src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue20372.cs

```diff
@@ -0,0 +1,26 @@
+﻿#if IOS
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue20372 : _IssuesUITest
+	{
+		public override string Issue => "[iOS] HTML Label not applying Bold or Italics on iOS";
+
+		public Issue20372(TestDevice device) : base(device)
+		{
+		}
+
+		[Test]
+		public void BothHtmlLabelsShouldApplyBoldAndItalicaProperties()
+		{
+			_ = App.WaitForElement("label1");
```

Comment:
This test is failing on iOS in this line, could you review it? Let me know if can help with something.

Reply:
Fixed